libs/SalesforceKit/SalesforceKit.modulemap

```diff
@@ -0,0 +1,54 @@
+framework module SalesforceKit {
+    umbrella header "SalesforceKit.h"
+    
+    export *
+    module * { export * }
+
+    framework module SalesforceSDKCore {
+        umbrella header "SalesforceSDKCore.h"
+
+        export *
+        module * { export * }
+    }
+
+    framework module SalesforceNetwork {
+        umbrella header "SalesforceNetwork.h"
+        
+        export *
+        module * { export * }
+        
+        use SalesforceSDKCore
+    }
+    
+    framework module SalesforceRestAPI {
+        umbrella header "SalesforceRestAPI.h"
+        
+        export *
+        module * { export * }
+        
+        use SalesforceSDKCore
+        use SalesforceNetwork
+    }
+
+    framework module SmartStore {
+        umbrella header "SmartStore.h"
+
+        export *
+        module * { export * }
+
+        use SalesforceRestAPI
+    }
+
+    framework module SmartSync {
+        umbrella header "SmartSync.h"
+        
+        export *
+        module * { export * }
+        
+        use SalesforceRestAPI
+    }
+
```

Comment:
I didn't think the HybridSDK was usable outside of something like PhoneGap/Cordova.  In that case, wouldn't the templates be used directly instead of explicitly adding something like SalesforceKit?

SalesforceReact seemed similar to me in that regard.